api/pkg/transformer/feast/feature_retriever.go

```diff
@@ -290,6 +305,21 @@ func (fr *FeastRetriever) newFeastCall(
 	}
 }
 
+func (fr *FeastRetriever) getFeastClient(endpoint string) feast.Client {
+	client, ok := fr.feastClients[Endpoint(endpoint)]
+	if !ok {
```

Comment:
I think this will hide issue if users incorrectly set the feast serving URL. They will create request to the default feast serving.

Reply:
I prefer to:
1. Check if the value is empty, if so use default URL
2. Check if the value exists, if not then throw error
3. Return existing client